listener/go.mod

```diff
@@ -5,11 +5,10 @@ go 1.19
 require (
 	github.com/go-logr/logr v1.2.3
 	github.com/go-logr/zapr v1.2.3
+	github.com/prometheus/client_golang v1.12.2
```

Comment:
why introduce Prometheus golang? is it directly used somewhere in code?

Reply:
I don't know TBH. `go mod tidy` did that.

Reply:
ah, we have metrics implemented for the watcher listener, make sense